packages/pools/src/errors.ts

```diff
@@ -10,6 +10,6 @@ export class NotEnoughLiquidityError extends Error {
   constructor() {
     super("Not enough liquidity");
     // Set the prototype explicitly.
-    Object.setPrototypeOf(this, NotEnoughLiquidityError.prototype);
```

Comment:
This should still be 

`Object.setPrototypeOf(this, NotEnoughLiquidityError.prototype);`

Otherwise prototype will not match.